src/sass/app_2/_common/components/range_slider.scss

```diff
@@ -142,57 +142,62 @@
     }
     &__line {
         position: absolute;
-        top: 9px;
+        top: 11px;
         left: 0;
         height: 2px;
-        background: $COLOR_BLUE;
+        background: $COLOR_LIGHT_GRAY_1;
         pointer-events: none;
         transition: width 0.5s;
     }
     &__caption {
         display: flex;
         padding: 4px 0;
         justify-content: space-between;
+        line-height: 18px;
     }
 }
 
+.sidebar-container fieldset div.range-slider__caption span {
+    font-size: 0.75rem;
```

Comment:
Will this rule not work if you move `span` inside the `&__caption` block above?

Reply:
no, it gets over written, so I have to either add !important or override it with specificity